src/amo/components/AddonBadges/styles.scss

```diff
@@ -12,9 +12,13 @@
 }
 
 .AddonBadges .Badge {
+  margin: 0 0 0 6px;
```

Comment:
This won't work when the language is RTL, so you'll want to use our RTL-aware mixin instead: `@include margin(0 0 0 6px);`.

Though in this case if you're only setting left margin you can just use `@include margin-start(6px);`

Reply:
Thanks for the suggestion, I have just updated it